frontend/src/components/UI/TabPanels/Tabs.vue

```diff
@@ -157,6 +159,130 @@
 						</template>
 					</Modal>
 				</q-tab-panel>
+
+				<q-tab-panel name="childs" class="flex q-gutter-md">
+					<div
+						class="cards_wrapper"
+						v-for="child in workspaceChilds"
+						:key="child.ID"
+					>
+						<div v-if="child.Logo">
+							<ChildsTabCard
+								v-if="child.Name"
+								:id="child.ID"
+								:parentID="child.ParentID"
+								:name="child.Name"
+								:shortDescription="child.ShortDescription"
+								:description="child.Description"
+								:logo="child.Logo"
+								:repositoryURL="child.GitURL"
+								@deleteProductAction="confirmDeleteChild(child.ID)"
+							/>
+						</div>
+						<div v-else>
+							<ChildsTabCard
+								v-if="child.Name"
+								:id="child.ID"
+								:parentID="child.ParentID"
+								:name="child.Name"
+								:shortDescription="child.ShortDescription"
+								:description="child.Description"
+								:logo="logo"
+								:repositoryURL="child.GitURL"
+								@deleteProductAction="confirmDeleteChild(child.ID)"
+							/>
+						</div>
+					</div>
+					<div class="panel_add__btn q-pa-md q-gutter-sm absolute-bottom-right">
+						<q-btn
+							color="white"
+							text-color="primary"
+							icon="add"
+							class="text-primary"
+							label="New child"
+							@click.prevent="openCreationProductModal(productTeam)"
+						/>
+					</div>
+					<!-- Childs Creation dialog -->
+					<Modal class="modalGlobal" v-model="isCreationProductsOpened">
```

Comment:
Export this modal in a component

Reply:
Modal is already a component with slots and templates --->  import Modal from "components/UI/Dialogs/Modal.vue";